### PR TITLE
add token value comparer to TokenBuilder

### LINKED ISSUE #111: tokenbuilder for token(column) <= ?
Hi,

I'd like to create the queries used in https://www.scylladb.com/2017/03/28/parallel-efficient-full-table-scan-scylla/ (source code: https://github.com/scylladb/scylla-code-samples/tree/master/efficient_full_table_scan_example_code) via the gocqlx query builder.

As far as I understand, neither the `TokenBuilder` nor the `Cmp` type supports creating a query like
`SELECT ... WHERE token(column) <= ?` and later on binding an int64 token value to the placeholder.

Did I miss something? if not, would you be interested in a PR?

### PR DESCRIPTION
**Pull request by zwopir**
fixes #111

This PR adds the token value compare functions to the `TokenBuilder` as discussed in #111

### CODE REVIEW

**Comment by zwopir** [comment]
travis now passes... ready for review

File: qb/token.go [hunk 1]
```diff
@@ -78,3 +128,14 @@ func (t TokenBuilder) cmp(op op, names []string) Cmp {
 		value:  Fn("token", s...),
 	}
 }
+
+func (t TokenBuilder) valueCmp(op op, name string) Cmp {
+	if name == "" {
+		name = "token_val"
```
Comment (mmatczuk) [inline qb/token.go]: maybe just `token`?

Comment (zwopir) [reply qb/token.go]: yes, done

**Comment by mmatczuk** [comment]
Thanks!

**Comment by zwopir** [comment]
you're welcome. Thanks for reviewing and merging

**Pull request merged**